apps/duellen/app/components/WikiLink.jsx

```diff
@@ -47,19 +47,18 @@ export default class WikiLink extends React.Component{
           <div className="w-100">
             {this.state.wikiresponse.map(({title, description}) => (
               <button key={title} onClick={() => this.props.onClick(title)} type="button" className="w-100 p-3 mb-2 bg-light text-black border">
-                  <p>
                     <b>{title}</b>
+                    {/* Test if it should only show the title
```

Comment:
What's this test? 😀

Reply:
Steffen wanted to test with some users if it's better with just the title

Reply:
Should we remove the commented stuff?